src/pymodaq_plugins_signal_recovery/daq_move_plugins/daq_move_Lockin_DSP7265.py

```diff
@@ -0,0 +1,273 @@
+                        "A"
+                    ).keys())
+                )
+            elif param.value() == DSP7265ThreadSafe.IMODES[2]:
+                self.settings.child('sensitivity').setLimits(
+                    list(build_dict_from_float_list(
+                        [s * DSP7265ThreadSafe.SEN_MULTIPLIER[2]
+                         for s in DSP7265ThreadSafe.SENSITIVITIES],
+                        "A"
+                    ).keys())
+                )
+        elif param.name() == "reference":
+            self.controller.reference = param.value()
+        elif param.name() == "fet":
+            self.controller.fet = FET[param.value()]
+        elif param.name() == "shield":
+            self.controller.shield = SHIELD[param.value()]
+        elif param.name() == "coupling":
+            self.controller.coupling = COUPLING[param.value()]
+        elif param.name() == "time_constant":
+            self.controller.time_constant = TIME_CONSTANTS[param.value()]
+        elif param.name() == "sensitivity":
+            if self.settings.child('imode').value() == DSP7265ThreadSafe.IMODES[0]:
+                self.controller.sensitivity = build_dict_from_float_list(
+                    [s * DSP7265ThreadSafe.SEN_MULTIPLIER[0]
+                     for s in DSP7265ThreadSafe.SENSITIVITIES],
+                    "V"
+                )[self.settings.child('sensitivity').value()]
+            if self.settings.child('imode').value() == DSP7265ThreadSafe.IMODES[1]:
+                self.controller.sensitivity = build_dict_from_float_list(
+                    [s * DSP7265ThreadSafe.SEN_MULTIPLIER[1]
+                     for s in DSP7265ThreadSafe.SENSITIVITIES],
+                    "A"
+                )[self.settings.child('sensitivity').value()]
+            if self.settings.child('imode').value() == DSP7265ThreadSafe.IMODES[2]:
+                self.controller.sensitivity = build_dict_from_float_list(
+                    [s * DSP7265ThreadSafe.SEN_MULTIPLIER[2]
+                     for s in DSP7265ThreadSafe.SENSITIVITIES],
+                    "A"
+                )[self.settings.child('sensitivity').value()]
+        elif param.name() == "voltage":
+            self.controller.voltage = param.value()
+        elif param.name() == "gain":
+            self.controller.gain = param.value()
+        else:
+            pass
+
+    def ini_stage(self, controller: object = None) -> edict:
+        """Actuator communication initialization
+
+        Parameters
+        ----------
+        controller: (object)
+            custom object of a PyMoDAQ plugin (Slave case). None if only one
+            actuator by controller (Master case)
+
+        Returns
+        -------
+        info: str
+            Id of the lockin
+        initialized: bool
+            False if initialization failed otherwise True
+        controller: object
+            Controller of the daq_move
+        """
+        try:
+            self.status.update(
```

Comment:
look at a recent version of the template for more concise way of doing this initialization